### PR TITLE
ui: fix default domainid for add account

### PR DESCRIPTION
**Pull request by shwstppr**
### Description

Fixes setting domainid param default value which was being set to 0.

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)
- [ ] build/CI

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### Screenshots (if appropriate):
Before:
Screenshot from 2024-01-03 18-32-13

After:
Screenshot from 2024-01-03 18-32-55

### CODE REVIEW

**Comment by shwstppr** [comment]
@blueorangutan ui

**Review by DaanHoogland — APPROVED** [review]
clgtm, the qa link doesn't work for me, any idea on that @shwstppr ?

**Comment by shwstppr** [comment]
@DaanHoogland it is working for me. For you is the page to opening or login not working or the fix not working? Incognito window may help

**Review by rohityadavcloud — APPROVED** [review]

File: ui/src/views/iam/AddAccount.vue [hunk 1]
```diff
@@ -303,7 +303,6 @@ export default {
         } else {
           this.loadMore(apiToCall, page + 1, sema)
         }
-        this.form.domainid = 0
```
Comment (rohityadavcloud) [inline ui/src/views/iam/AddAccount.vue:306]: not sure if it's applicable here - there's also a way to delete this.form.domainid if the key/value need to be removed from the object.

**Review by harikrishna-patnala — APPROVED** [review]
LGTM

verified with https://qa.cloudstack.cloud/simulator/pr/8435

File: ui/src/views/iam/AddAccount.vue (hunk 1 above)
Comment (shwstppr) [reply ui/src/views/iam/AddAccount.vue:306]: @rohityadavcloud don't think we need to remove domainid from the object as API call will be just adding to the existing list so previous selection will still be there

**Pull request merged**